### DISCUSSION TITLE
AttributeError : h11 defaut_headers missing

### DISCUSSION THREAD

**Discussion by gitthious** [emmett-framework/emmett#450] [Q&A]
Hi, I have just discovered Hemmett. I'm working actually on py 3.11. So I tried on it, just with helloword with a separate conda env. unfortunalty, I got followinf error :

>   File "C:\Users\Demo\miniconda3\envs\testEmmett\lib\asyncio\selector_events.py", line 826, in _read_ready__data_received
>     self._protocol.data_received(data)
>   File "C:\Users\Demo\miniconda3\envs\testEmmett\lib\site-packages\uvicorn\protocols\http\h11_impl.py", line 183, in data_received
>     self.handle_events()
>   File "C:\Users\Demo\miniconda3\envs\testEmmett\lib\site-packages\emmett\asgi\protocols\http\h11.py", line 167, in handle_events
>     default_headers=self.default_headers,
> AttributeError: 'H11Protocol' object has no attribute 'default_headers'

Then I retried with py 3.7. The same.

I'm on windows 10 with firefox 106.0.2 (64 bits). I have searched a couple of hours to read code and tried some tips. Nothing ! Have you already found this problem ? Have you an idea ?

I am dispointed because  the reasons why you create Hemmett touched me

Thank you for your time. Best regards

**Comment by gi0baro** [comment]
Hi @gitthious, I just released 2.4.12 that should fix your issue.

Please, mind that Python 3.11 support is still targeted for 2.5 release, which is expected to be available in the next months.
